src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskCounter = 0;
+
+    public static void printHorizontalLine() {
+        System.out.println("    ——————————————————————————————————————————————————");
+    }
+
+    public static void printTasks(int taskCounter, ArrayList<Task> tasks) {
+        for (int i=0; i<taskCounter; i++) {
```

Comment:
Perhaps you can separate with some spaces like so to improve readability:
`for (int i = 0; i < taskCounter; i++) {`